relay-server/src/envelope.rs

```diff
@@ -41,14 +41,13 @@ use failure::Fail;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use smallvec::SmallVec;
 
-use relay_common::DataCategory;
 use relay_general::protocol::{EventId, EventType};
 use relay_general::types::Value;
 use relay_sampling::TraceContext;
 
 use crate::constants::DEFAULT_EVENT_RETENTION;
 use crate::extractors::{PartialMeta, RequestMeta};
-use crate::utils::{infer_event_category, ErrorBoundary};
+use crate::utils::ErrorBoundary;
```

Comment:
`infer_event_category` used to be private before #931, so we can remove the `pub` there. It was intended as a private function, really.

Reply:
Done.